core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config.jelly

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2020, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="variables" title="${%variablesLabel}"
+             description="${%variablesDescription}" >
+        <f:textbox value="${instance.variables}" />
+    </f:entry>
+    <f:entry field="retainCharacteristicEnvVars" title="${%retainCharacteristicEnvVarsLabel}">
```

Comment:
the checkbox isn't associated to the label by putting the title on the entry (which means you can't click on the label and have it checked, required as part of accessibility), from my understanding it should go on the checkbox, cc @jsoref @fqueiruga

Reply:
Right.

Reply:
Old habits 😅 

But it looks a bit weird; perhaps better with the redesign PR… thoughts?

<img width="1259" alt="Screenshot" src="https://user-images.githubusercontent.com/1831569/80034471-c0cd8a00-84ee-11ea-962e-054cea570584.png">

Reply:
"the sooner the redesign lands the less time it'll look weird."

Reply:
Can you try adding a label on the entry as well? maybe call it Retention or something like that

Reply:
There are two different entries here. Do you want a label only for the first one? That seems to abuse the current form structure for a group that doesn't usually exist.

Perhaps hide these behind Advanced button?

Reply:
Sorry didn't see your reply, maybe that was a good idea

Reply:
@daniel-beck and @fqueiruga is this one resolved now?

Reply:
As this concerns general UI only it can easily be improved upon after this PR is merged, so would prefer this not hold up the entire change.

(Unless someone can demonstrate that the options don't make sense as checkboxes, of course, which might impact the underlying model.)